sub.go

```diff
@@ -95,7 +95,8 @@ type SubscriptionPendingInvoiceItemIntervalParams struct {
 
 // SubscriptionTransferDataParams is the set of parameters allowed for the transfer_data hash.
 type SubscriptionTransferDataParams struct {
-	Destination *string `form:"destination"`
+	AmountPercent *float64 `form:"amount_percent"`
```

Comment:
I don't understand this change at all. It's already in master https://github.com/stripe/stripe-go/blob/master/sub.go#L230-L234
I don't understand what's going on with the diff here.